src/api.ts

```diff
@@ -7,9 +7,14 @@ function wait(delay: number) {
   return new Promise(resolve => setTimeout(resolve, delay));
 }
 
-export async function getPeople(): Promise<Person[]> {
-  // keep this delay for testing purpose
+export function getPeople(): Promise<Person[]> {
   return wait(500)
     .then(() => fetch(API_URL))
-    .then(response => response.json());
+    .then(response => {
+      if (!response.ok) {
+        throw new Error();
```

Comment:
It's better to throw a more descriptive error. For example: 'throw new Error(`Failed to fetch people data: ${response.statusText}`);'